src/runtime-tools/win64/onefuzz.ps1

```diff
@@ -168,3 +170,16 @@ function Install-VCRedist {
   Start-Process -FilePath C:\onefuzz\vcredist_x86.exe -ArgumentList "/install /q /norestart" -Wait -WindowStyle Hidden
   log "installing VC Redist: done"
 }
+
+function Install-Dotnet {
+  log "Installing dotnet"
+  &powershell -NoProfile -ExecutionPolicy unrestricted -Command "[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12; &([scriptblock]::Create((Invoke-WebRequest -UseBasicParsing 'https://dot.net/v1/dotnet-install.ps1'))) -Version 6.0.300 -InstallDir c:\onefuzz\tools\dotnet"
```

Comment:
We don't really need to use the one-liner version of this, right? That's probably more a convenience for users from the docs. It also has downsides: it's sort fussy to obvious parameters to this function (`$Version`, `$InstallDir`) without having to worry about string interpolation stuff.

Can we just explode this into multiple lines, and maybe parameterize the containing function so the end result gets used like:
```powershell
Install-Dotnet -Version $env:DOTNET_VERSION -InstallDir $env:DOTNET_ROOT
```